src/WebUI/src/pages/characters/[id]/index.vue

```diff
@@ -131,10 +132,55 @@ onBeforeRouteUpdate(async to => {
 });
 
 await fetchPageData(character.value.id);
+
+//
+
+const wrapper = ref<ComponentPublicInstance<typeof VOnboardingWrapper> | null>(null);
+const { start, goToStep, finish } = useVOnboarding(wrapper);
+const steps = computed(() => {
+  return [
+    {
+      attachTo: {
+        element: '[data-aq-onboarding-level]',
+      },
+      content: {
+        title: 'Level',
+        description:
+          'Lorem ipsum, dolor sit amet consectetur adipisicing elit. Voluptatem porro repellendus voluptatum ea quis veniam dignissimos quos reiciendis enim non.',
```

Comment:
Your level represents your progress and growth of your character as you gain experience through playing cRPG, as you accumulate experience points (XP), your character's level increases, resulting in higher usable Attribute, Skill and Weapon Proficiency Points.